CHANGELOG.md

```diff
@@ -1,21 +1,75 @@
+## 0.7.0
+
+- Calculate component dependencies from stack imports
+- Added `deps` output for each component
+- Reduced unnecessary stack runs in CI/CD systems like Spacelift
+
+- The provider already calculates these parameters:
+    - `imports` - a list of all imports (all levels) for the stack
+    - `stacks` - a list of all stacks in the infrastructure where the component is defined
+
+- Both `imports` and `stacks` are not 100% suitable to correctly determine the YAML config files that a component depends on
+    - `imports` is too broad. The provider returns all direct and indirect imports for the stack, even those that don't define any variables for the
+      component. This will trigger the component's stack in Spacelift even if the unrelated imports are modified resulting unrelated stack runs in
+      Spacelift
+    - `stacks` is too broad and too narrow at the same time. On the one hand, it detects all stacks in the infrastructure where the component is
+      defined, but we don't need to trigger a particular stack in Spacelift if some other top-level YAML stack configs are modified. On the other
+      hand, it misses the cases where a YAML stack config file specifies global variables, which are applied to the component as well
+
+BACKWARDS INCOMPATIBILITIES / NOTES:
+
+## 0.6.0
+
+- Updates github release action to use 1.16
+
+BACKWARDS INCOMPATIBILITIES / NOTES:
+
+## 0.4.2
+
+- Added `process_stack_deps` input var to the stack_config data source (configurable, `false` by default) .Not all provider invocations need to
+  process all stack dependencies for the components (e.g. Spacelift module needs it, remote-state does not). Makes invocations without processing
+  stack dependencies 2-3 times faster
```

Comment:
Bulleted items do not need to end with periods if they are only one sentence (or sentence fragment), but if they are multiple sentences and you use a period at the end of the other sentences, you should use a period at the end of the last sentence, too.
```suggestion
  stack dependencies 2-3 times faster.
```